app/tutorials/highway-management/src/main/java/org/eclipse/mosaic/app/tutorial/communication/RoundTripMessage.java

```diff
@@ -0,0 +1,42 @@
+package org.eclipse.mosaic.app.tutorial.communication;
```

Comment:
This class misses a license header

Reply:
Sorry, my browser view somehow hide the license header..

Reply:
https://github.com/eclipse/mosaic/pull/15/commits/096c216d551346206a2bb78603d705c411619c34
I thought I added it with this commit. 🤨

Reply:
You did, everything's fine